sdk/communication/communication-short-codes/src/utils/constants.ts

```diff
@@ -1,4 +1,4 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export const SDK_VERSION: string = "1.0.0-beta.1";
+export const SDK_VERSION: string = "1.0.0-beta.2";
```

Comment:
consider updating package.json with this: https://github.com/Azure/autorest.typescript/wiki/%60core-http%60-dependency-migration-to-%60core-client%60-%60core-rest-pipeline%60#metadata-in-packagejson

example: https://github.com/Azure/azure-sdk-for-js/pull/20375/files#diff-fb830fc560d86afb8a6bd6e99072e7dc0aa91d0325a37cb1e882556052ab4fa4R39